Integrations/integration-VirusTotal.yml

```diff
@@ -234,13 +241,30 @@ script:
                             Scans: scansTable,
                             ScanID: r[i].scan_id
                         }
-                    }
+                    };
                     ec[outputPaths.file] = scans_ec;
                 }
             } else {
-                ec[outputPaths.file][0]['VirusTotal'] = {
-                    ScanID: r[i].scan_id,
-                    MD5: r[i].md5
+                if (ec[outputPaths.file]){
+                    scans_ec = {
+                        ScanID: r[i].scan_id
+                    };
+                    if (typeof ec[outputPaths.file][0] === 'object') {
+                        ec[outputPaths.file][0].VirusTotal = scans_ec;
```

Comment:
add some comments, otherwise it is hard to understand what you is happening here